Prototope/Screen.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  Screen.swift
+//  Prototope
+//
+//  Created by Jason Brennan on Jun-16-2015.
+//  Copyright (c) 2015 Khan Academy. All rights reserved.
+//
+//	Swifty re-write of https://github.com/adamwulf/ios-uitouch-bluedots
+
+import UIKit
+import UIKit.UIGestureRecognizerSubclass
+
+
+/** Type representing the Screen of the device. */
+public struct Screen {
+	
+	public static var touchDotsEnabled: Bool = false {
+		didSet {
+			if touchDotsEnabled {
+				enableTouchDots()
+			} else {
+				removeTouchDots()
+			}
+		}
+	}
+	
+	private static var overlayView: TouchDotOverlayView? = nil
+	private static func enableTouchDots() {
+		overlayView = TouchDotOverlayView(frame: CGRect(Layer.root.frame))
+		let window = UIApplication.sharedApplication().delegate?.window
+		window!!.addSubview(overlayView!)
```

Comment:
The double-force is a little weird! Maybe it'd be better to use `delegate!` on the previous line—I think that makes your intent clearer (i.e. "this code assumes that the delegate is non-nil and also that the delegate's window is non-nil")